README.md

```diff
@@ -1,6 +1,49 @@
 # DHIS2 Workflows for GitHub Actions
 
+This repository provides a centralized location for GitHub Actions workflows that can be utilized across various DHIS2 repositories.
+
 ## Directory structure:
+**ci**: Solutions for Continuous Integration.
+
+**automation**: Solutions for automating workflows.
+## Consuming the Workflows:
+Workflows in this repository can be utilized in other repositories by either copying them or by calling them directly if they support the `workflow_call` event.
+
+### Consuming the e2e-prod Workflow:
+The `e2e-prod` workflow provides an end-to-end testing solution, running Cypress E2E tests against various production versions of DHIS2.
+
+#### Requirements:
+To utilize this workflow:
+
+In the DHIS2 repository where you wish to implement this workflow, create a new workflow file (e.g., `.github/workflows/use-e2e-prod.yml`).
+Use the following template to call the central `e2e-prod` workflow:
+
+```yaml
+name: Use e2e-prod
+
+on: [push, pull_request]  # or any other GitHub event
+
+jobs:
+    e2e-tests:
+        uses: dhis2/workflows/.github/workflows/cypress-e2e-tests.yml@main
+        secrets:
+            baseurl: ${{ secrets.BASEURL }}
```

Comment:
I'm not sure about this BASEURL too.. @jenniferarnesen do you know what should the value of `CYPRESS_DHIS2_BASE_URL_DEV`  should look like here https://github.com/dhis2/data-visualizer-app/blob/41d8557192ceba255fa8ae7782c55cf88f439cb2/.github/workflows/dhis2-verify-app.yml#L84 ?
btw maybe it could be defined as a variable (not a repo secret )

Reply:
Jennifer said the `baseurl` value should be `https://test.e2e.dhis2.org/analytics-dev`
I update the doc with this example